yorkie/src/main/kotlin/dev/yorkie/util/ConnectRpcErrorUtil.kt

```diff
@@ -0,0 +1,22 @@
+package dev.yorkie.util
+
+import com.connectrpc.Code
+import com.connectrpc.ConnectErrorDetail
+import com.connectrpc.ConnectException
+
+/**
+ * `isRetryable` will return true if the given error is retryable.
+ */
+public fun isRetryable(exception: ConnectException?): Boolean {
+    if (exception == null) {
+        return false
+    }
+
+    val errorCode = exception.code
+    val isRetryableErrorCode = errorCode === Code.CANCELED ||
```

Comment:
should it check for instance equality instead of object equality?

Reply:
As you say, `==` is a better way for comparison. 
So I've made the function more readable including your suggestion.